app/controllers/feedbacks_controller.rb

```diff
@@ -61,6 +62,19 @@ def set_feedback
     authorize @feedback
   end
 
+  def create_redirect
+    if session[:request_url].include?('last_submitted_hours_and_feedbacks')
+      if @feedbackable.class == Assignment
+        redirect_to last_submitted_hours_and_feedbacks_assignment_path(@feedbackable), make_notice
+      else
+        group_assignment = @feedbackable.group_assignments.where(volunteer: @volunteer).last
+        redirect_to last_submitted_hours_and_feedbacks_group_assignment_path(group_assignment), make_notice
```

Comment:
It would be easier to do:
```ruby
@feedbackable = @feedbackable.group_assignments.where(volunteer: @volunteer).last if @feedback.group_offer?
polymorphic_path(@feedbackable, action: :last_submitted_hours_and_feedbacks)

Reply:
or maybe another option to get `GroupOffer` type paths:

```ruby
GroupAssignment.find_by(group_offer: @feedbackable, volunteer: @volunteer)
```

Reply:
Oh, actually you'd need to use where and .last, because in case there is more than one `GroupAssignment` matching, you'd want to grab the later.

Reply:
Maybe I am a bit confused, because I don't see why that is simpler.